x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -430,6 +487,36 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
               data-test-subj="mlAnalyticsCreateJobFlyoutDestinationIndexInput"
             />
           </EuiFormRow>
+          <EuiFormRow
+            label={i18n.translate('xpack.ml.dataframe.analytics.create.excludedFieldsLabel', {
+              defaultMessage: 'Excluded fields',
+            })}
+            helpText={i18n.translate('xpack.ml.dataframe.analytics.create.excludedFieldsHelpText', {
+              defaultMessage:
+                'Optionally select fields to be excluded from analysis. All other supported fields will be included',
+            })}
+          >
+            <EuiComboBox
+              aria-label={i18n.translate(
+                'xpack.ml.dataframe.analytics.create.excludesInputAriaLabel',
+                {
+                  defaultMessage: 'Optional. Enter or select field to be excluded.',
+                }
+              )}
+              isDisabled={isJobCreated}
+              isLoading={loadingFieldOptions}
+              options={excludesOptions}
+              selectedOptions={excludes.map(field => ({
+                label: field,
+              }))}
+              onCreateOption={onCreateOption}
+              onChange={selectedOptions =>
```

Comment:
Consider moving callback init before render. Using an arrow function in render creates a new function each time the component renders, which may break optimizations based on strict identity comparison.

Reply:
Good call - moved here 5fb02c50cabc18a94c4952adee9d279bea7b54ac